libraries/botbuilder-core/tests/conversationState.test.js

```diff
@@ -64,23 +64,15 @@ describe(`ConversationState`, function() {
         }
     });
 
-    it(`should throw install exception if get() called without a cached entry.`, function(done) {
```

Comment:
@stevengum Is this a bug in the code or just a test that needs deleting?

`conversationState` is a `Map()`, so calling `get()` on an item that doesn't exist returns `undefined` instead of throwing an error. The `assert(false...` is the error that actually gets thrown and then ignored by the `catch`

I think changing the SDK so it throws might be a back-compat issue, so maybe just delete the test?

Reply:
@mdrichardson this test can be removed, I'm not sure why it exists...